### ISSUE TITLE
Any Documents or Tutorials?

### ISSUE THREAD

**Issue by zjxeditor** [GPUOpen-LibrariesAndSDKs/RadeonProRender-Baikal#181]
Hi, I wonder if there is any documents or tutorials for this cool open source rendering engine. I feel it is pretty hard to dive into the source code. In additions, where can I find some details of the material model and the light model? Then, I am curious about the BaikalDataGenerator. What is it? How can I use it?

**Comment by zjxeditor** [comment]
And I think it better to provide a detailed sample to demonstrate how to add lights, how to change materials and how to render the "Science Fiction" scene just like the picture in the README file.

**Comment by harokyang** [comment]
Read the test cases
They covered almost everything you need

**Comment by zjxeditor** [comment]
Yes, the test cases are good start points. But it is really necessary to read the source code to understand the framework and API usage, then you can do you own customization.
